rosie/core/__init__.py

```diff
@@ -28,7 +28,10 @@ def __init__(self, settings, adapter):
         self.settings = settings
         self.dataset = adapter.dataset
         self.data_path = adapter.path
-        self.suspicions = self.dataset[self.settings.UNIQUE_IDS].copy()
+        if self.settings.UNIQUE_IDS:
+            self.suspicions = self.dataset[self.settings.UNIQUE_IDS].copy()
+        else:
+            self.suspicions = self.dataset.copy()
```

Comment:
that's why I thought combining those 3 columns might help... but it wasn't enough, I believe we need a brainstorm to figure this one out... So far I'm good with having all the columns in the suspicions file :)